indico/web/client/js/custom_elements/tips.scss

```diff
@@ -0,0 +1,112 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+@use 'design_system';
+
+ind-with-tooltip,
+ind-with-toggletip {
+  --tooltip-surface-color: #000;
```

Comment:
Would it cause any a11y problems if we changed this to `#333`? Feels a bit very dark right now.

![image](https://github.com/indico/indico/assets/179599/96b172be-10ef-4cfa-b1d3-9733691b7e91)
![image](https://github.com/indico/indico/assets/179599/d926b874-04e9-4a8b-b876-4eea8e1c0b0b)


(Might also be personal preference, but for anything dark-mode-related I generally prefer the "dim" version over the "full black background" version, and I guess this also applies here even if it's just a tooltip)

Reply:
At least right now we don't have anything darker than the menus and the meeting page background....

(so far the preference within the team is on the `#333` version btw)

Reply:
Pushed